test/infrastructure/docker/docker/loadbalancer.go

```diff
@@ -137,6 +140,10 @@ func (s *LoadBalancer) IP(ctx context.Context) (string, error) {
 	if err != nil {
 		return "", errors.WithStack(err)
 	}
+	if lbip4 == "" {
+		// if there is a load balancer container with the same name exists but is stopped, it may not have IP address associated with it.
+		return "", errors.Errorf("LoadBalancer IP cannot be empty: container %s does not have an associated IP address", s.containerName())
```

Comment:
```suggestion
		// if there exists a load balancer container with the same name but is stopped, it may not have an IP address associated with it.
		return "", errors.Errorf("load balancer IP cannot be empty: container %s does not have an associated IP address", s.containerName())
```
nit: golang errors usually start with lower case

Reply:
@ashish-amarnath sorry for the nitpicking the space between `load` and `balancer` was intended for consistency as we're using this variant in l.99